.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+**Ticket:** [#](TRELLOURL)
+**Intent:**
+**How to test (optional):**
+
+### All of the following commands should pass.
+
+- [ ] `npm test`
```

Comment:
I wonder if it's more convenient to add some pre-commit-hooks and/or set up Github actions/Travis as a separate task for this instead?

Reply:
If so, might as well remove this file altogether?